frontend/src/App.jsx

```diff
@@ -1,22 +1,49 @@
 import MapComponent from './map';
 import EditMenu from './edit';
-import React, { useState } from 'react';
+import React, { useState, useEffect } from 'react';
 
-export default function App () {
+import { getAllScenarios } from './requests';
 
+export default function App() {
   const [editMenuIsOpen, setEditMenuIsOpen] = useState(false);
+  const [parcelCoords, setParcelCoords] = useState(null);
+  const [parcelID, setParcelID] = useState(null);
+  const [savedScenarios, setSavedScenarios] = useState(null);
 
   const toggleEditMenu = () => {
     setEditMenuIsOpen((prev) => !prev);
-  }
+  };
+
+  useEffect(async () => {
+    const scenarios = await getAllScenarios();
+    setSavedScenarios(scenarios);
+  }, []);
+
+  const refreshSavedScenarios = async () => {
+    const scenarios = await getAllScenarios();
+    setSavedScenarios(scenarios);
+  };
 
-  console.log('render', editMenuIsOpen);
   return (
     <div className="App">
       <div className="map-and-menu-container">
-        <MapComponent toggleEditMenu={toggleEditMenu} />
-        <EditMenu open={editMenuIsOpen} />
+        <MapComponent
+          toggleEditMenu={toggleEditMenu}
+          setParcelCoords={setParcelCoords}
+          setParcelID={setParcelID}
+        />
+        {(savedScenarios)
+          ? (
+            <EditMenu
+              open={true}
+              parcelCoords={parcelCoords}
+              parcelID={parcelID}
+              refreshSavedScenarios={refreshSavedScenarios}
+              savedScenarios={savedScenarios}
+            />
+          )
+          : <div />}
```

Comment:
Hmm, I'm not sure I can explain that behavior. But this did need fixing. It's okay to render the component with no saved scenarios. We just needed to initialize `savedScenarios` as `[]` instead of `null`.

Reply:
Side note, is there an easy way to clear the scenario store during development?

Reply:
Easy-ish yeah, 
![Capture](https://user-images.githubusercontent.com/4071255/174386583-13baf37e-a2e8-40c7-8736-443ee9583c65.PNG)

Reply:
I think the Firefox inspect pane doesn't have the same feature, but I'll find another way to do it.